### ISSUE TITLE
Add date

### ISSUE THREAD

**Issue by davidlmobley** [livecomsjournal/article_templates#10]
We should add a date display in the template so all PDFs are automatically dated, e.g.:
```
% For ongoing development use
\date{\today}
% For release versions use
%\date{ January 20, 2017 }
```

**Comment by davidlmobley** [comment]
@liantze - any chance we could get this added? I was thinking in the left column just below the blurb about GitHub. Probably it could say, "This version dated November 23, 2017" or similar.

Thanks!

**Comment by liantze** [comment]
Yep, just submitted a PR for this.

**Comment by davidlmobley** [comment]
Closed via #40
